test/integration/SchemaDesign/SchemaHandler.test.js

```diff
@@ -44,7 +44,7 @@ describe('Actions', () => {
     let graknTx = await graknSession.transaction().write();
     const schemaHandler = new SchemaHandler(graknTx);
 
-    await schemaHandler.defineAttributeType({ attributeLabel: 'name', dataType: 'string' });
+    await schemaHandler.defineAttributeType({ attributeLabel: 'name', valueType: 'string' });
```

Comment:
why ared the mocked value types `String` and this `string`?

Reply:
good catch, josh!
resolved in https://github.com/graknlabs/workbase/pull/291/commits/dc170df40bb42ce67da51f5b885a710e1cedecfd